sdk/core/core-util/review/core-util.api.md

```diff
@@ -17,6 +17,14 @@ export interface AbortOptions {
     abortSignal?: AbortSignalLike;
 }
 
+// @public
+export function calculateExponentialDelayInterval(retryAttempt: number, config: {
+    retryDelayInMs: number;
+    maxRetryDelayInMs: number;
```

Comment:
Decided not to introduce a separate type for this, but let me know if you disagree!